.github/workflows/build_windows_pytorch_wheels.yml

```diff
@@ -134,7 +152,10 @@ jobs:
       - name: Configure MSVC
         uses: ilammy/msvc-dev-cmd@0b201ec74fa43914dc39ae48a89fd1d8cb592756 # v1.13.0
 
+      # Checkout main sources from https://github.com/pytorch/pytorch
+      # TODO: switch to 'nightly' to match our Linux workflows?
```

Comment:
Probably should, right? It'd provide a good A/B datapoint in case a change breaks Windows but not Linux?